nx/include/switch/services/fs.h

```diff
@@ -249,6 +269,9 @@ Result fsDeleteSaveDataFileSystemBySaveDataSpaceId(FsSaveDataSpaceId saveDataSpa
 
 Result fsIsExFatSupported(bool* out);
 Result fsOpenGameCardFileSystem(FsFileSystem* out, const FsGameCardHandle* handle, FsGameCardPartiton partition);
+Result fsReadSaveDataFileSystemExtraDataBySaveDataSpaceId(void* buf, size_t len, FsSaveDataSpaceId saveDataSpaceId, u64 saveID);
```

Comment:
Shouldn't these use the above struct?

Reply:
I was hesitant to do that because it implies the commands only accept a fixed size struct. That may not necessarily always be the case, especially given the large amount of uninitialized data it contains at the end. N may potentially use buffers less than the full size, so I wanted to keep the option there to preserve that behaviour (should anything use it).

Reply:
ok